pavelib/quality.py

```diff
@@ -50,8 +50,7 @@ def find_fixme(options):
         apps_list = ' '.join(top_python_dirs(system))
 
         pythonpath_prefix = (
-            "PYTHONPATH={system}:{system}/lib"
-            "common/djangoapps:common/lib".format(
+            "PYTHONPATH={system}:common/djangoapps:common/lib".format(
```

Comment:
YES, good catch.  I changed that, and it fixed "Unable to import 'celery.states'" violations.  Now it complains about other aspects of celery, but I can fix that in another PR.